src/components/Book.jsx

```diff
@@ -17,22 +23,39 @@ const Book = ({ book }) => {
     return null; // or render a loading state
   }
   return (
-    <div>
-      <div key={book.item_id}>
-        <h3>{book.title}</h3>
-        <p>
-          Author:
-          {' '}
-          {book.author}
-        </p>
-        <p>
-          Category:
-          {' '}
+    <div key={book.item_id} className="book-card">
+      <div className="book-detail">
+        <p className="book-category">
           {book.category}
         </p>
-        <button type="button" onClick={handleRemoveBook}>
-          Remove
-        </button>
+        <span className="book-title">
+          {book.title}
+        </span>
+        <span className="author">{book.author}</span>
+        <div className="book-actions-container">
+          <button className="comments" type="button">Comments</button>
+          <div className="actions-separate-line" />
+          <button className="remove" type="button" onClick={handleRemoveBook}>
+            Remove
+          </button>
+          <div className="actions-separate-line" />
+          <button className="edit" type="button">Edit</button>
+        </div>
+      </div>
```

Comment:
- [ ] Great job @mulugithub by implementing the changes as I requested previously but it seems like you messed up with your design, which makes your design is now more `inconsistent` and looks really unprofessional. Please have a look at the attached image and table and don't forget to align the items correctly.

### Requirement
![Screenshot from 2023-09-09 21-59-56](https://github.com/mulugithub/bookstore/assets/66476329/dbfd71d2-7a0d-47df-911c-07801c432843)

### Design Comparison
<table>
<tr>
<th>Zeplin's Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-09-09 22-02-19](https://github.com/mulugithub/bookstore/assets/66476329/3aaf1a69-224b-4a24-b6b4-c435c97f6ee6)

</td>
<td>

![image](https://github.com/mulugithub/bookstore/assets/66476329/5e9f05cf-0ade-4172-9adf-f1597a43884f)

</td>
</tr>
</table>

Reply:
**Dear CR,**

_Thank you for reviewing my project_

I completed all the given requirements. 

- [ ] I observed not much need for mobile-first design in this milestone. Even the feedback is confusing. Kindly provide me with the details of the feedback on my project as outlined in the requirement.

Reply:
### Check it again

### **Given design**

![image](https://github.com/mulugithub/bookstore/assets/61753075/39c71b54-f46e-491a-9b5b-6b25de697369)
### 
 **my project screenshot:** 

![image](https://github.com/mulugithub/bookstore/assets/61753075/27a260dc-fa39-40e3-9a68-37ef4ea43c47)

Reply:
- [ ] Dear @mulugithub, As suggested by the previous reviewer, your design seems to be suffering from a layout shift. currently, this is how your design looks on a laptop display. Your layout becomes a bit messy for any screen size <1400px. Remember as a common practice, the desktop vies start from 1024px, and we need to consider these screen sizes.

### Your design on a screen size <1400px
![Screen Shot 2023-09-09 at 11 06 51 PM](https://github.com/mulugithub/bookstore/assets/69077061/744ade15-e08c-4aec-88d6-9c264046190c)

Reply:
As suggested by the previous reviewer, your desktop screen display still have some fallout for screen size 1024, kindly implement the change I highlighted with yellow pen in the screenshots below :

![image](https://github.com/mulugithub/bookstore/assets/20537712/53ee9b99-ef99-4677-bb68-3ac9b919df9a)

![image](https://github.com/mulugithub/bookstore/assets/20537712/6760456f-12d4-48b2-96a3-67597b20cda7)